metadata-ingestion/sql-etl/common.py

```diff
@@ -99,7 +100,7 @@ def produce_dataset_mce(mce, kafka_config):
     record_schema = avro.load(kafka_config.avsc_path)
     producer = AvroProducer(conf, default_value_schema=record_schema)
 
-    producer.produce(topic=kafka_config.kafka_topic, value=mce)
+    producer.produce(topic=kafka_config.kafka_topic, key=str(uuid4()), value=mce)
```

Comment:
Actually the best value for `key` would be the URN specified in the mce instead of a random number.

Reply:
Ok, I have done some modification.